crates/gen-spidermonkey/src/lib.rs

```diff
@@ -48,191 +48,191 @@ lazy_static! {
             WasmSignature {
                 params: vec![],
                 results: vec![],
-                retptr: false,
+                retptr: false,indirect_params:false,
```

Comment:
Looks like this file has some bad formatting, probably from conflict resolution?

Reply:
Oh this is me using a random macro and praying rustfmt will fix everything for me when, in fact, it did not.